content/md/first-steps/manage-your-attributes.md

```diff
@@ -251,6 +213,63 @@ Like on `Settings/Rules`, you can perform several actions from this grid:
 - Execute rules
 - Delete rules
 - Calculate the number of affected products by the rule
+# List attributes
+
+To start, go to `Settings > Attributes`. From this page, you will have several options to work with the datagrid of attributes as described below.
+
+## Attribute grid pagination
+
+The attribute grid displays 25 attributes per page. To browse attribute's pages, click on the page number you want to reach.
+
+![image](../img/Settings_Attributes.png)
+
+## Search on the attribute label
+
+To quickly find an attribute, you can use the search bar to search on the attribute label.
+
+![image](../img/Settings-UsetheSearchBar.gif)
+
+::: warning
+If your attribute does not have any label translated yet, it will not pop in the search results. Make sure your attribute has its labels translated!
+:::
+
+## Filter on attributes
+
+The attribute datagrid comes with a filter bar.
+
+![image](../img/Settings_AttributesFilters.png)
+
+1.  Enter your filter criteria
+1.  The grid is automatically updated when a new filter is set
+
+You can filter on the following attribute information:
+- The attribute type
+- The attribute group it belongs to
+- The “scopable” property (one value per channel)
+- The “localizable” property (one value per locale)
+
+::: ee
+You can also filter on the property "smart". The attribute is smart if there is a rule defined on this attribute.
+:::
+
+::: info
+Labels filters are localised: If an attribute has a label in the locale preference, then this wording is used. If there is no available language, then the attribute code will be displayed in brackets. For example: [COLOR].
```

Comment:
I'll do two different sentences here.
"Labels filters are localised. If an attribute has a label in the locale preference, then this wording is used."

Reply:
I don't understand "If there is no available language"